pkg/storer/internal/upload/uploadstore.go

```diff
@@ -572,7 +572,7 @@ func Report(ctx context.Context, st transaction.Store, chunk swarm.Chunk, state
 			return nil
 		}
 
-		return fmt.Errorf("failed to read uploadItem %s: %w", ui, err)
+		return fmt.Errorf("failed to read uploadItem %x: %w", ui.BatchID, err)
```

Comment:
Maybe too late now that it has been merged, but the full key to an uploadItem (according to line 563) is the chunk address AND the batchID.  But I didn't check if upstream invokers of `Report()` might add the chunk ID to their logs if this error ever actually occurs.